main/src/ca/uwaterloo/flix/language/errors/ResolutionError.scala

```diff
@@ -734,6 +734,25 @@ object ResolutionError {
     }
   }
 
+  /**
+   * An error raised to indicate that a static field name was not found.
+   *
+   * @param clazz the class name.
+   * @param field the field name.
+   * @param loc   the location of the field access.
+   */
+  case class UndefinedJvmStaticField(clazz: Class[_], field: Name.Ident, loc: SourceLocation) extends ResolutionError with Recoverable {
```

Comment:
The Boolean `static` for the `UndefinedJvmField` is no longer used, right?

Reply:
It _is_ used. But the whole could should go away once we have field support.